test/core/data.wast

```diff
@@ -412,6 +412,12 @@
   "type mismatch"
 )
 
+(module
+  (global (export "global-i32") i32 (i32.const 0))
+  (global (export "global-mut-i32") (mut i32) (i32.const 0))
+)
+(register "test")
+
```

Comment:
This shouldn't be needed. Validation has to happen before linking, so it mustn't matter for any of the tests whether the imported module actually exists.

Same with the other tests.